core/SConscript.unix

```diff
@@ -415,16 +401,6 @@ SOURCE_UNIX = [
     'vendor/micropython/shared/runtime/gchelper_generic.c',
 ]
 
-if NEW_RENDERING:
-    SOURCE_MOD += [
-        'embed/trezorhal/unix/display_driver.c',
-        'embed/trezorhal/xdisplay_legacy.c',
-    ]
-else:
-    SOURCE_MOD += [
-        'embed/trezorhal/unix/display-unix.c',
```

Comment:
this file should be deleted

Reply:
deleted, see 0b1c85d7a7e21a583c521d8061a32f0b1901a0ca